src/Annotations/Serializer/CommonMark/functions_include.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+// Don't redefine the functions if included multiple times.
+if (!function_exists('eLife\Annotations\Serializer\CommonMark\clean_paragraph')) {
+    require __DIR__.'/functions.php';
```

Comment:
use `require_once`?

Reply:
What is the problem in referring directly to `functions.php` in the autoload definition?

Reply:
I haven't experienced the issue myself but saw this:
https://github.com/guzzle/psr7/pull/22

Reply:
Not sure whether it's an issue for us or not; in theory, we should only include once the autoloader with `require_once 'vendor/autoload.php`, but that has to be verified.

Reply:
I'll get rid of it. If we ever experience an issue we can reintroduce